api/coreservice.go

```diff
@@ -350,6 +404,7 @@ func (core *coreService) Account(addr address.Address) (*iotextypes.AccountMeta,
 		accountMeta.ContractByteCode = code
 	}
 	span.AddEvent("bc.BlockHeaderByHeight")
+	tipHeight := core.bc.TipHeight()
```

Comment:
the `tipHeight` may be from a different value from `state` read from https://github.com/iotexproject/iotex-core/pull/4339/files#diff-a84e0d7fdb009bebf88eb3c2582dc4c2f5fce09f6d40a3a514b553c198675f42R373